docs/functions/usage/index.md

```diff
@@ -89,7 +89,7 @@ function App() {
 
   useEffect(() => {
     functions()
-      .httpsCallable('listProducts')()
+      .httpsCallable('listProducts')({ abc: 123 })
```

Comment:
This change is still present, I'm proposing a suggestion and will commit it - the rest looks great now

```suggestion
      .httpsCallable('listProducts')()
```